### PR TITLE
Add a link hijacker component

### PR DESCRIPTION
**Pull request by schwers-zz**
👓  @phil303

### CODE REVIEW

File: lib/modules/components.js [hunk 1]
```diff
@@ -142,6 +152,93 @@ export const Anchor = connect(null, anchorDispatcher)(_Anchor);
...
+    dangerouslySetInnerHTML: T.object,
+  }
+
+  static defaultProps = {
+    navigateToPage: () => {},
+    onClick: () => {},
+  };
+
+  extractValidPath($link) {
+    let href = $link.getAttribute('href');
```
Comment (phil303) [inline lib/modules/components.js]: looks like this can be a const

Comment (schwers-zz) [inline lib/modules/components.js]: good catch

**Comment by schwers-zz** [comment]
💇 , switched to a node-less higher order component

**Comment by schwers-zz** [comment]
Added some updates to readme, will squash that and the other commits later if we go with this

**Comment by phil303** [comment]
:fish:

**Pull request merged**